src/auto/coverage.rs

```diff
@@ -4,21 +4,24 @@
 
 use glib::translate::*;
 use pango_sys;
+use std::fmt;
 use std::mem;
 use std::ptr;
 use CoverageLevel;
 
 glib_wrapper! {
-    #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
-    pub struct Coverage(Shared<pango_sys::PangoCoverage>);
+    pub struct Coverage(Object<pango_sys::PangoCoverage, CoverageClass>);
 
     match fn {
-        ref => |ptr| pango_sys::pango_coverage_ref(ptr),
-        unref => |ptr| pango_sys::pango_coverage_unref(ptr),
+        get_type => || pango_sys::pango_coverage_get_type(),
```

Comment:
Oh this sucks. Before 1.44 the get_type function was not available so we handled it as shared type, now it's there and we handle it as proper object.

This is such a special case it should be handled manually, it makes no sense to implement in gir.

Reply:
Done